examples/boilerplate-react/src/typings.d.ts

```diff
@@ -6,10 +6,10 @@
  */
 
 import * as React from 'react';
-import { IntrinsicTelements } from '@telements/components-web/dist/IntrinsicTelements';
+// import { IntrinsicTelements } from '@telements/components-web/dist/IntrinsicTelements';
```

Comment:
Will you uncomment these lines in the future?

Reply:
yes, because we currently have 2 web-component setups for exploration
1. Lit-Html (commented out)
2. Stencil

In the PR description `Create separate demos for stencil and lit-html` refers to this point. We need to decide which one to use, and abandon the other...

Reply:
ok